biobuddy/components/generic/biomechanical_model.py

```diff
@@ -123,15 +122,22 @@ def to_real(self, data: Data, gravity: Point = None) -> BiomechanicalModelReal:
                 )
 
                 for marker in segment.markers:
-                    with error_context(f"Marker '{marker.name}'"):
+                    try:
                         model.segments[marker.parent_name].add_marker(marker.to_marker(data, model, scs_global))
+                    except Exception as e:  # Marker try
```

Comment:
If you revert the "error_context" do not forget to expand the error message as "The marker 'name'" is not enough (it needs to add which segment for instance, this was automatically done previously but not anymore)